app/graphql/types/image.rb

```diff
@@ -29,4 +29,8 @@ def views(names: object.styles.keys)
       }
     end
   end
+
+  field :blurhash, String,
+    null: true,
+    description: 'A blurhash-encoded version of this image'
```

Comment:
```suggestion
    description: 'A blurhash-encoded version of this image.'
```

Reply:
Bitch no periods

Reply:
Fine, we going no periods for any single line periods?